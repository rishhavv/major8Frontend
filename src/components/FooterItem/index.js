import { Typography } from 'antd'
import React from 'react'

const FooterItem=({title,data})=>{
    data.map((item,key)=>{
        console.log("ite is ",item.value)
    })
    return(
        <div style={{flex:1,alignItems:'flex-start'}}>
            <Typography style={{fontSize:20 ,alignItems:'flex-start',color:'white'}}>
                {title}
            </Typography>
            <div style={{paddingTop:10}}>
                {
                    data.map((item,key)=>{
                        return(<Typography style={{color:'white'}}>
                            {item.value}
                            </Typography>
                        )
                        
                    })
                }
                
            </div>
        </div>
    )
}

export default FooterItem