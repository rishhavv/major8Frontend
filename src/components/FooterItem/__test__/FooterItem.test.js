import {cleanup, fireEvent, render} from '@testing-library/react';
import FooterItem from '../index'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

afterEach(cleanup)
const Company=[
    {
        value:'About',
        link:'about'
    },
    {
        value:'Legal',
        link:''
    },
    {
        value:'Privacy Policy',
        link:''
    },
    {
        value:'Terms Of Service',
        link:''
    },
    {
        value:'Contact Us',
        link:''
    },

]
it('should render footer correctly',()=>{
    const wrapper=shallow(
        <FooterItem
        title='Company'
        data={Company}
        />
    )

    expect(wrapper).toMatchSnapshot()
})