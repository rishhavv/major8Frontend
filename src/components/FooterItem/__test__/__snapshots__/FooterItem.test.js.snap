// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `ShallowWrapper {}`;

exports[`should render data correctly 1`] = `ShallowWrapper {}`;

exports[`should render footer correctly 1`] = `ShallowWrapper {}`;
